aspnetcore/web-api/advanced/formatting.md

```diff
@@ -95,30 +95,50 @@ services.AddMvc(options =>
 
 If your application needs to support additional formats beyond the default of JSON, you can add NuGet packages and configure MVC to support them. There are separate formatters for input and output. Input formatters are used by [Model Binding](xref:mvc/models/model-binding); output formatters are used to format responses. You can also configure [Custom Formatters](xref:web-api/advanced/custom-formatters).
 
-### Adding XML Format Support
+::: moniker range=">= aspnetcore-3.0"
 
-To add support for XML formatting, install the `Microsoft.AspNetCore.Mvc.Formatters.Xml` NuGet package.
+### Configure System.Text.Json-based formatters
 
-Add the XmlSerializerFormatters to MVC's configuration in *Startup.cs*:
-
-[!code-csharp[](./formatting/sample/Startup.cs?name=snippet1&highlight=2)]
-
-Alternately, you can add just the output formatter:
+Features for the `System.Text.Json`-based formatters can be configured using <xref:Microsoft.AspNetCore.Mvc.MvcOptions.SerializerOptions>.
```

Comment:
Is `SerializerOptions` a new API in 3.0? I ask because this xref link doesn't work.

Reply:
Yes, this would be new in 3.0.

Reply:
I've removed the link until the 3.0 APIs have been published.